CPP_STYLE_GUIDE.md

```diff
@@ -316,13 +311,19 @@ exports.foo = function(str) {
 
 #### Avoid throwing JavaScript errors in nested C++ methods
 
-When you have to throw the errors from C++, try to do it at the top level and
+When you need to return an error from C++, try to do it at the top level and
```

Comment:
This referred specifically to throwing exceptions, though.

(s/error/exception/ would be a good change here, that’s on me.)

Reply:
ack

Reply:
Can you clear this up for me? we can't c++ `throw`, we need to `isolate()->ThrowException()` right?